fix: update openshift doc with default scc (#1894)
nginx/documentation: content/nim/deploy/kubernetes/deploy-using-helm.md

Use the file with the `-f values.yaml` flag when installing the chart.

{{< call-out "note" "OpenShift support" >}}

> OpenShift support was added in NGINX Instance Manager 2.19. To enable it, add the setting `openshift.enabled: true` to your `values.yaml` file. Starting with NGINX Instance Manager v2.22, NGINX Instance Manager supports the default SCC on OpenShift.

Reviewer: ```suggestion
OpenShift support is available in NGINX Instance Manager 2.19 and later. To enable it, add `openshift.enabled: true` to your `values.yaml` file. In NGINX Instance Manager 2.22 and later, NGINX Instance Manager supports the default Security Context Constraints (SCC) on OpenShift.
```

## Appendix: OpenShift security constraints {#appendix-openshift-security-constraints}

> OpenShift restricts containers from running as root by default. When you enable OpenShift support in the NGINX Instance Manager Helm chart:

Reviewer: ```suggestion
OpenShift restricts containers from running as root by default.

To enable OpenShift support in the NGINX Instance Manager Helm chart, set the following value:

```

```yaml
openshift:
  enabled: true
```

> NGINX Instance Manager (NIM) is deployed on OpenShift. Starting with NIM v2.22.0, the chart no longer creates or requires a custom [Security Context Constraints (SCC)](https://docs.redhat.com/en/documentation/openshift_container_platform/4.15/html/authentication_and_authorization/managing-pod-security-policies) with fixed UIDs (**1000** for `nms`, and **101** for `nginx` and `clickhouse`). Instead, NGNIX Instance Manager runs using the **default SCC** applied by OpenShift to the NGINX Instance Manager service account.

Reviewer: ```suggestion
With this setting enabled, NGINX Instance Manager deploys on OpenShift. In NGINX Instance Manager 2.22.0 and later, the chart no longer creates or requires a custom [Security Context Constraints (SCC)](https://docs.redhat.com/en/documentation/openshift_container_platform/4.15/html/authentication_and_authorization/managing-pod-security-policies))). Instead, NGINX Instance Manager runs with the default SCC that OpenShift applies to the NGINX Instance Manager service account. The chart no longer uses fixed UIDs (**1000** for `nms`, and **101** for `nginx` and `clickhouse`).
```
